resources/views/admin/pages/product/form.blade.php

```diff
@@ -66,13 +66,17 @@
             </span>
             <div class="mt-2">
                 <label class="inline-flex items-center text-gray-600 dark:text-gray-400">
-                    <input type="radio" {{ old('status', isset($product) ? $product->status : null) == '0' ? 'checked' : '' }} class="text-purple-600 form-radio focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:focus:shadow-outline-gray" name="status" value="0" />
+                    <input type="radio" {{ old('status', isset($product) ? $product->status : null) == 'active' ? 'checked' : '' }} class="text-purple-600 form-radio focus:border-purple-400 focus:outline-none focus:shadow-outline-purple dark:focus:shadow-outline-gray" name="status" value="active" />
                     <span class="ml-2">Pending</span>
```

Comment:
why use static `active` word? We don't want to use static that's why the given task is to use enum.